src/sbml2symbolics.jl

```diff
@@ -0,0 +1,31 @@
+using ModelingToolkit, Symbolics
+
+allowed_funs = Dict(
+    "+" => :+,
+    "-" => :-,
+    "*" => :*,
+    "/" => :/,
+    #TODO add translations of SBML functions to Julia
+)
+
+allowed_sym(x) = haskey(allowed_funs,x) ? allowed_funs[x] : DomainError(x,"Unknown SBML function")
+
+function SBML2Symbolics(x::Math)
```

Comment:
there's the standard way of converting stuff in julia, I'd overload Base.convert

Reply:
fixed with 6913135